ci-operator/step-registry/openshift/e2e/test/qe/destructive/openshift-e2e-test-qe-destructive-chain.yaml

```diff
@@ -0,0 +1,9 @@
+chain:
+  as: openshift-e2e-test-qe-destructive
+  steps:
+  - ref: openshift-extended-test-destructive
```

Comment:
@liangxia no need to create new step openshift-extended-test-destructive and just use openshift-extended-test with setting FILTER.

we ever create new golang upgrade step because it has different parameter with openshift-extended-test.
For openshift-extended-test  and openshift-extended-test-destructive, they are same logic and just different value for parameter.